docs/create-packages/creating-a-package-dotnet-cli.md

```diff
@@ -18,55 +18,51 @@ This topic applies only to .NET and other projects that use the [SDK-style forma
-- `Company` is the company name. If not specified, the default value is the package ID.
+The `pack` command also uses the following properties. If you don't supply values in the project file, the command uses default values.
 
-In Visual Studio, you can set these values in the project properties. Right-click the project in Solution Explorer, select **Properties**, and then select the **Package** section. You can also set these properties directly in the *.csproj* or other project file.
+- `Version` is a specific version number in the form `Major.Minor.Patch[-Suffix]`, where `-Suffix` identifies [prerelease versions](prerelease-packages.md). If not specified, the default value is `1.0.0`.
+- `Authors` are the authors of the package. If not specified, the default value is the `AssemblyName`.
+- `Company` is company information. If not specified, the default value is the `Authors` value.
+- `Product` is product information. If not specified, the default value is the `AssemblyName`.
 
-```xml
-<PropertyGroup>
-  <PackageId>Unique.Name</PackageId>
-  <Version>1.0.0</Version>
-  <Authors>your_name</Authors>
-  <Company>your_company</Company>
-</PropertyGroup>
-```
+In Visual Studio, you can set these values in the project properties. Right-click the project in **Solution Explorer**, select **Properties**, and then select the **Package** section. You can also add the properties directly to the *.csproj* or other project file.
 
-You can create a new default project by using the `dotnet new classlib` command. The following example shows the project file with the required package properties added.
+The following example shows a project file with package properties added.
 
 ```xml
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <PackageId>AppLogger</PackageId>
+    <PackageId>UniqueName</PackageId>
     <Version>1.0.0</Version>
-    <Authors>your_name</Authors>
-    <Company>your_company</Company>
+    <Authors>My Name</Authors>
+    <Company>Contoso</Company>
   </PropertyGroup>
 </Project>
 ```
 
-You can add optional properties, such as `Title`, `PackageDescription`, and `PackageTags`. You can also surface assets from dependencies directly in the package by using the `IncludeAssets` and `ExcludeAssets` attributes.
+You can add other optional properties, such as `Title`, `PackageDescription`, and `PackageTags`.
+
+>[!NOTE]
+> For packages you build for public consumption, pay special attention to the `PackageTags` property. Tags help others find your package and understand what it does.
+
+The `dotnet pack` command automatically converts `PackageReference`s  in your project files to dependencies in the created package. You can control which assets to include through the `IncludeAssets` and `ExcludeAssets` tags. For more information, see [Controlling dependency assets](../consume-packages/package-references-in-project-files.md#controlling-dependency-assets).
```

Comment:
```suggestion
The `dotnet pack` command automatically converts `PackageReference`s  in your project files to dependencies in the created package. You can control which assets to include through the `IncludeAssets`, `ExcludeAssets` and `PrivateAssets` tags. For more information, see [Controlling dependency assets](../consume-packages/package-references-in-project-files.md#controlling-dependency-assets).
```

Reply:
For context, PrivateAssets is something that controls whether a packagereference becomes a dependency in a resulting package, so calling it out makes it complete.